lacework-agent/values.schema.json

```
@@ -66,6 +66,96 @@
             ],
             "additionalProperties": false
         },
+        "clusterAgent": {
+            "type": "object",
+            "required": [
+                "image",
+                "enable",
+                "clusterType",
+                "scrapeInitialDelayMins",
+                "scrapeIntervalMins"
```

Why are these scrape times required? Their type declarations below include `null`.

If `null` is allowed then remove them from required.

If they are required, provide values in `values.yaml`.
```suggestion
                "clusterType"
```

---

CC : @anilnanduri @rnalexlacework